src/webgpu/api/validation/encoding/programmable/pipeline_bind_group_compat.spec.ts

```diff
@@ -693,3 +697,79 @@ g.test('empty_bind_group_layouts_requires_empty_bind_groups')
       encoder.finish();
```

Comment:
Comment below reminded me that we also need a test or test case for `dispatchWorkgroupsIndirect()`.

Reply:
ok, let me also prepare a test for `dispatchWorkgroupsIndirect()`.